integration/testdata/schema/schema.bubbly

```diff
@@ -185,3 +185,38 @@ table "tempo_periods" {
         }
     }
 }
+
+table "snyk_vulnerabilities" {
+    field "id" {
+        type = string
+        unique = false
+    }
+    field "name" {
+        type = string
+        unique = false
+    }
+    field "creationTime" {
+        type = string
+        unique = false
+    }
+    field "cvssScore" {
+        type = string
+        unique = false
+    }
+    field "language" {
+        type = string
+        unique = false
+    }
+    field "severity" {
+        type = string
+        unique = false
+    }
+    field "version" {
+        type = string
+        unique = false
+    }
+    field "cve" {
+        type = string
+        unique = false
+    }
+}
```

Comment:
Do (optional) clauses `unique = false` add value?

Reply:
No and they are no longer required. Could remove them

Reply:
I forgot that those were optional. Good point, will remove.